src/components/Item/Item.js

```diff
@@ -1,24 +1,60 @@
 // @flow
-import * as React from 'react';
+import React, { Component } from 'react';
 import './Item.css';
 import ProgressBar from '../ProgressBar/ProgressBar';
 
-const Item = ({ item }) => (
-  <article id={item.id}>
-    <h2>{item.name}</h2>
-    <dl>
-      <dt>Mål:</dt>
-      <dd>{item.goal}</dd>
-      <dt>Användningar:</dt>
-      <dd>{item.delimiter}</dd>
-      <dt>Inköpspris:</dt>
-      <dd>{item.price}</dd>
-    </dl>
-    <p>
-      Framsteg<br />
-      <ProgressBar progressBarMax={item.goal} progressBarCurrent={item.delimiter} />
-    </p>
-  </article>
-);
+class Item extends Component {
+  constructor(props) {
+    super(props);
+
+    // This binding is necessary to make `this` work in the callback
+    this.toggleCard = this.toggleCard.bind(this);
+
+    this.state = { isOpen: false };
+  }
+
+  toggleCard = () => {
+    this.setState(prevState => ({
+      isOpen: !prevState.isOpen,
+    }));
+  };
+
+  render() {
+    const { item } = this.props;
+
+    return (
```

Comment:
I've tested out the component and the toggling works out :smile_cat:
If we only wrap the `<dl>` in a conditional statement, we'll get away with less repetition of code :dancing_women: 

```jsx
<article id={item.id} onClick={this.toggleCard}>
        <h2>{item.name}</h2>
        {
          this.state.isOpen && (
            <dl>
              <dt>Mål:</dt>
              <dd>{item.goal}</dd>
              <dt>Användningar:</dt>
              <dd>{item.delimiter}</dd>
              <dt>Inköpspris:</dt>
              <dd>{item.price}</dd>
            </dl>
          )
        }
        <p>
              Framsteg<br />
          <ProgressBar
            progressBarMax={item.goal}
            progressBarCurrent={item.delimiter}
          />
        </p>
      </article>
```